.yarn/sdks/typescript/lib/typescript.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+const {existsSync} = require(`fs`);
+const {createRequire, createRequireFromPath} = require(`module`);
+const {resolve, dirname} = require(`path`);
+
+const relPnpApiPath = "../../../../.pnp.js";
+
+const absPnpApiPath = resolve(__dirname, relPnpApiPath);
+const absRequire = (createRequire || createRequireFromPath)(absPnpApiPath);
```

Comment:
I noticed that we are repeating this in a couple of places... maybe we should make it reusable?

Reply:
Ignore all inside of `sdks`. I'll add it to `.gitignore`, I think those should be downloaded via vcode.

Reply:
If you're using PnP it should be commited, if not you can just remove it :)

Reply:
Thanks really appreciated feedback. How this deal with conflicts? I guess eventually the code might be updated.

Reply:
When you update Yarn you'd also run `yarn dlx @yarnpkg/pnpify --sdk vscode` to update it, but unless something changes from vscode you most likely wont have to.